FSharpLu.Json/Compact.fs

```diff
@@ -10,18 +13,33 @@ open Microsoft.FSharp.Reflection
 type CompactUnionJsonConverter() =
     inherit Newtonsoft.Json.JsonConverter()
 
+    let processWithContractResolver (serializer:JsonSerializer) (name:string) =
+        let toPascal (name:string) =
+            if System.Char.IsUpper (name, 0) then name
+            else string(System.Char.ToUpper name.[0]) + name.Substring(1)  
+        let toCamel (name:string) =
+            if System.Char.IsLower (name, 0) then name
+            else string(System.Char.ToLower name.[0]) + name.Substring(1)    
+        match serializer.ContractResolver with 
+        | :? CamelCasePropertyNamesContractResolver -> toCamel name
+        | :? DefaultContractResolver -> toPascal name
+        | _ -> name
+
     let SomeFieldIdentifier = "Some"
 
+    let stringEq (a:string) (b:string) =
```

Comment:
Yes, in order to inline a `let` binding you need to make it a top-level binding in the module. But there is no drawback in doing this.